packages/lib/server/username.ts

```diff
@@ -0,0 +1,141 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import slugify from "@calcom/lib/slugify";
+import prisma from "@calcom/prisma";
+
+import notEmpty from "../../../apps/website/lib/utils/notEmpty";
+import { wordlist } from "../../../apps/website/lib/utils/wordlist/wordlist";
```

Comment:
This will break cloning for al community contributors as they don't have access to these files.